packages/api-folders/src/types.ts

```diff
@@ -0,0 +1,178 @@
+import { TenancyContext } from "@webiny/api-tenancy/types";
+import { Context as BaseContext } from "@webiny/handler/types";
+import { I18NContext } from "@webiny/api-i18n/types";
+import { SecurityContext, SecurityIdentity } from "@webiny/api-security/types";
+
+export interface CreatedBy {
+    id: string;
+    type: string;
+    displayName: string | null;
+}
+
+export enum Type {
+    PAGE = "page",
+    CMS = "cms",
+    FILE = "file"
+}
```

Comment:
The issue with this is that you're hardcoding these 3 values, which implicitly makes this package aware of _where_ it's going to be used. Let's not use enums for this; if we do, we won't be able to use this `api-folders` with any other app, unless we add a new enum value.